src/pages/PassportRedemption/PassportVoucher.tsx

```diff
@@ -13,7 +13,8 @@ import BackImage from './VoucherBack.png';
 
 const Voucher = () => {
   const { id, tickets_secret } = useParams();
-  let params = new URLSearchParams(useLocation().search);
+  const location = useLocation().search;
```

Comment:
@chrisng93 can you verify that this is all right? I think is weird rebase shows these as changes even though this is the current state of the world already.